catalogue_pipeline/transformer/src/main/scala/uk/ac/wellcome/transformer/source/SierraMaterialType.scala

```diff
@@ -0,0 +1,3 @@
+package uk.ac.wellcome.transformer.source
+
+case class SierraMaterialType(code: String, value: String)
```

Comment:
There's a similar field "Country" – only on the Sierra API, no prefix. I'm namespacing it as part of the language transformation; we should be consistent.

Reply:
I'm not sure whether these qualify as the same type even though they have the same fields.

Reply:
Right, I was unclear.

My suggestion was to move it into the "sierra" namespace, then rename it "MaterialType".